apps/fabric-website/src/pages/Overviews/ResourcesPage/docs/default/ResourcesDesignResources.md

```diff
@@ -3,10 +3,12 @@
 These design toolkits provide styles, controls and layout templates that enable you to create beautiful and coherent Microsoft experiences.
 
 <ul class="md-list--flex">
-  <li class="mdut--half">Web Fluent (Sketch): Coming soon</li>
-  <li class="mdut--half">Web Fluent (Figma): Coming soon</li>
-  <li class="mdut--half">iOS Fluent (Sketch): Coming soon</li>
-  <li class="mdut--half">Android Fluent (Sketch): Coming soon</li>
+  <li class="mdut--half">[Web Fluent (Sketch)](https://microsoft.sharepoint-df.com/:u:/t/OPGUXLeads/EU75cB2smYVBrJRYdm1B02YBCfn-UR27avtbe4SWZDyrhg?e=kspHRE)</li>
```

Comment:
Can we host these on a CDN or something instead of SharePoint? I know these are guest links, but I'm still not sure it's appropriate to use SharePoint to host public content like this.

Reply:
@Jahnp Who do we need to talk to in order to get this approved, and will there be any consequences (ie. developer workload if they need to be updated) in re-hosting them?

Reply:
+1. @jordandrako I'll create an aka.ms link you can use here instead.

Reply:
Just to clarify, I think that even if we have aka.ms links, they should **not** point to content on SharePoint. The content should live somewhere else.